deny.toml

```diff
@@ -1,7 +1,5 @@
 [advisories]
 ignore = [
-    "RUSTSEC-2020-0014", # Advisory for dev dependency of `trust-dns` for testing
-                         # dns servers.
     "RUSTSEC-2020-0002", # tracked in #1639
     "RUSTSEC-2020-0001", # dev dependency only
```

Comment:
`RUSTSEC-2020-0001` not required with this changes too?
https://rustsec.org/advisories/RUSTSEC-2020-0001.html

Reply:
Yup, good catch we can remove that one as well!